### PR TITLE
FT transfer: Add remaining smaller components and style adjustments

### PR DESCRIPTION
**Pull request by Patrick1904**
_Effort part of https://github.com/near/near-wallet/issues/1714_

New components:
1. TransactionDetails.js
2. Information.js (entry type)
3. AccountId.js
4. AccountIdQRCode.js
5. SelectTokenButton.js

Modifications:
1. Moved TokenIcon.js out into its own file
2. Various style adjustments
3. Various translate-related naming changes

### CODE REVIEW

**Review by MaximusHaximus — CHANGES_REQUESTED** [review]
💖  These are looking awesome -- nice work here, easy to follow/read, and well factored components with small areas of responsibility. 💯 🏅 

Nothing major, but I added a few questions in-line :)

File: src/components/send/components/TransactionDetails.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            >
+                <Amount
+                    translateIdTitle={prefixTXEntryTitledId('estimatedFees')}
+                    amount='500000000000000000000'
+                    translateIdInfoTooltip='profile.security.mostSecureDesc'
+                />
+                <Amount
+                    translateIdTitle={prefixTXEntryTitledId('estimatedTotal')}
+                    amount='9900000000000000000000'
+                    translateIdInfoTooltip='profile.security.mostSecureDesc'
```
[the remark this answers is not in the document]
Comment (Patrick1904) [reply src/components/send/components/TransactionDetails.js:33]: The tooltip translate IDs are not correct - I've requested the correct copy for all the tooltips and will update once they're available.

File: src/translations/en.global.json [hunk 2]
```diff
@@ -1114,7 +1114,9 @@
             "insufficient": "You are attempting to send your entire available balance. At least <b>${data} NEAR</b> must be reserved to cover transaction fees."
         }
     },
-
+    "receiveV2": {
+        "accountIdTitle": "Account ID"
```
[the remark this answers is not in the document]
Comment (Patrick1904) [reply src/translations/en.global.json]: Totally, I forgot that I had added `input.accountId.title` before. Updated it.

**Review by MaximusHaximus — APPROVED** [review]
:+1: lets do it!

**Pull request merged**